src/LibraryView.tsx

```diff
@@ -61,6 +63,17 @@ export class LibraryView {
                 thisObject.layoutSpecsJson = JSON.parse(jsonString);
                 thisObject.updateContentsInternal();
             });
+
+        // fetch search items (in json format) if queryString exists and is not empty
+        if (queryString && queryString.length > 0) {
+            fetch(queryString)
```

Comment:
FETCH is awesome API. But it doesn't work on all the browsers.. just add a comment here with browser compatibility.